Engine/Component/ComponentBillboard.h

```diff
@@ -66,6 +70,7 @@ class ComponentBillboard : public Component
 	float current_sprite_x = 0, current_sprite_y = 0;
 	float sheet_speed = 1;
 	bool oriented_to_camera;
+	AnimationType animation_type = AnimationType::CONSTANT;
```

Comment:
For now not use but will be next thing to add

Reply:
In which cases you want a spritesheet animated random? If you use a spritesheet to be animated is because you want it in certain order right?

Reply:
I was looking at the fire tutorial where the flames must be random to simulate the fire.